Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8
+FROM fedora:32
```

Comment:
please use a fully qualified registry, so that we know where the image came from.

Reply:
Thanks